### PR TITLE
Inference integration test for autosegmentation of heart ventricle

### PR DESCRIPTION
**Pull request by aihsani**
- Added integration test for segmentation inference
- Made monai label server port configurable via `MONAILABEL_SERVER_PORT` environment variable
  - This is useful for local testing
- Fixed non-terminating server when integration tests fail

### CODE REVIEW

File: tests/integration/test_infer.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import unittest
+
+import requests
+
+from . import SERVER_URI
+
+
+class EndPointInfer(unittest.TestCase):
+    def test_segmentation(self):
+        model = "segmentation_left_atrium"
```
Comment (SachidanandAlle) [inline tests/integration/test_infer.py:10]: lets have gpu check.. and skip if gpu is not enabled kind..
```
        if not torch.cuda.is_available():
            return
```

Comment (aihsani) [reply tests/integration/test_infer.py:10]: why? this can happen on CPU even if it takes longer. it's just inference

Comment (SachidanandAlle) [reply tests/integration/test_infer.py:10]: K.. some models don't work due to specific network/op not available on cpu.. we can merge if this run for both

**Review by SachidanandAlle — APPROVED** [review]

File: tests/integration/test_infer.py (hunk 1 above)
Comment (aihsani) [reply tests/integration/test_infer.py:10]: I see. OK. Changing

**Pull request merged**